### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Mike-devloper**
Add popup window for mobile and desktop

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -563,3 +573,144 @@ input {
...
+  padding-bottom: 10px;
+}
+
+.poplinks {
+  height: auto;
+  width: 340px;
+  margin: 5% 0% 0% 5%;
+  gap: 5px;
+  display: flex;
+}
```
Comment (omarsalem7) [inline styles.css:678]: - [ ] Kindly fix the width of the buttons to match it like Figma, the following image in iphone6/7/8
>image

File: index.html [hunk 2]
```diff
@@ -160,7 +160,7 @@ <h3 class="work_title">Multi-Post Stories Gain+Glory</h3>
                     </div>
 
                         <form action="work.html">
-                            <button type="button" class="button2">See Project</button>
+                            <button type="button" class="button2 See_Project btn4">See Project</button>
                         </form>
```
Comment (omarsalem7) [inline index.html:164]: - [ ] Consider the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript. Also, consider using the same data for both works projects and models for them.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Mike-devloper 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Mike-devloper, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 
- Your code is clean and well documented ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (oguntayomicheal) [reply index.html]: Hello @omarsalem7, thanks for your review, it is really helpful. I will implement the suggested changes.

**Comment by oguntayomicheal** [comment]
Hello @ShadyShawkat, thanks for your review.

**Pull request merged**